sig-autoscaling/charter.md

```diff
@@ -0,0 +1,78 @@
+# SIG Autoscaling Charter
+
+This charter adheres to the conventions described in the [Kubernetes Charter README] and uses
+the Roles and Organization Management outlined in [sig-governance].
+
+## Scope
+
+Covers development and maintenance of components for automated scaling in
+Kubernetes.  This includes automated vertical and horizontal pod
+autoscaling, initial resource estimation, cluster-proportional system
+component autoscaling, and autoscaling of Kubernetes clusters themselves.
+
+### In scope
+
+- Autoscaling-related API objects, such as the HorizontalPodAutoscaler and
+  VerticalPodAutoscaler
+
+- Autoscaling-related tools, such as the cluster autoscaler,
+  single-component scaling tools (e.g. pod-nanny), and
+  cluster-proportional scaling tools
+
+- Ensuring API interfaces (the scale subresource) are available and usable
+  to enable other SIG to write autoscalable objects, and enable people to
+  interact with those interfaces.
+
+[Link to SIG section in sigs.yaml][sigs.yaml]
+
+#### Code, Binaries and Services
+
+- Components and utilities that take automated action to scale a component
```

Comment:
These are all kind of vague, pointers to actual repos/folders?

Reply:
I was under the impression that concrete folders is what sigs.yaml was supposed to be for, and that this was more of "conceptually, what do you cover".  That being said, I can certainly provide some examples here, or actually links if my impression was incorrection.

Reply:
fixed to mention the actual subproject names